README.md

```diff
@@ -78,6 +78,9 @@ In this example:
 - `dev` will use the `uv-venv-lock-runner` and use `uv sync` to install dependencies to the environment with the `dev`,
   `test` and `type` extra groups.
 
+Note that when using `uv-venv-lock-runner`, all dependencies will come from the lock file, controlled by `extras`.
```

Comment:
I think here we should specify that the only configuration other than core tox settings that are respected will be extras. All of the Python-specific configuration are ineffective within this type of run environment, including deps.

Reply:
I was trying to convey that with ``controlled by `extras`.`` – should I generalize the second sentence with something like "such as"? I suspect it's gonna be tough (and changing) to circle them perfectly.

Reply:
Yeah, I would like to try to generalize it. Because essentially nothing but dependencies is supported here.of the Python namespace So perhaps we should just call out to that.

Reply:
better?